src/dsp_tools/commands/xmlupload/stash/upload_stashed_resptr_props.py

```diff
@@ -31,25 +33,74 @@ def upload_stashed_resptr_props(
 
     print(f"{datetime.now()}: Upload the stashed resptrs...")
     logger.info("Upload the stashed resptrs...")
+
     upload_state.pending_stash = cast(Stash, upload_state.pending_stash)
     link_value_stash = cast(LinkValueStash, upload_state.pending_stash.link_value_stash)
-    for res_id, stash_items in link_value_stash.res_2_stash_items.copy().items():
-        res_iri = upload_state.iri_resolver.get(res_id)
+
+    _upload_process(link_value_stash=link_value_stash, iri_resolver=upload_state.iri_resolver, con=con, context=context)
+    _remove_empty_ids(link_value_stash=link_value_stash)
+
+
+def _upload_process(
+    link_value_stash: LinkValueStash,
+    iri_resolver: IriResolver,
+    con: Connection,
+    context: dict[str, str],
+) -> None:
+    """
+    Try to upload the stashed items. If the target_iri of the item can be found
+    and the item can then successfully be uploaded it is removed from link_value_stash.
+    The link_value_stash is updated accordingly, as a side effect.
```

Comment:
The sentence before says: "it is removed from link_value_stash". It doesn't make sense to repeat this information

```suggestion
```